jina/clients/base/http.py

```diff
@@ -150,6 +152,7 @@ def _result_handler(result):
                 request_handler=_request_handler,
                 result_handler=_result_handler,
                 logger=self.logger,
+                prefetch=prefetch or 0,
```

Comment:
same here, I believe it can handle `None`

Reply:
I believe it should be `preferch` or `self.args.prefetch` or `self.prefetch` for backwards compatibility?